joomla/joomla-cms
# [4.0] fix wc

Pull Request for Issue # .

### Summary of Changes


Webcomponents polyfill doesnt come with .min files, so we need an uglified version and this PR just does that

### Testing Instructions

Apply Patch and use Firefox or Edge to loagin to the backend. Check that alerts are showing

### Expected result



### Actual result



### Documentation Changes Required
